thirdeye/thirdeye-frontend/app/pods/aiavailability/route.js

```diff
@@ -93,13 +93,13 @@ export default Route.extend({
         anomaliesArray.forEach(anomalyId => {
           const row = flow;
           row['link to RCA'] = anomalyId;
-          row.comment = row.comment.anomalyId;
+          row.comment = row.comment[anomalyId];
           // build array of strings to put on DOM
           row.columns = [];
           tableHeaders.forEach(header => {
             const cellValue = row[header.text];
             if (header.text === 'link to RCA') {
-              const domString = `<a href="https://thirdeye.corp.linkedin.com/app/#/rootcause?anomalyId=${cellValue}">Comment of the anomaly</a>`;
+              const domString = `<a href="https://thirdeye.corp.linkedin.com/app/#/rootcause?anomalyId=${cellValue}">Investigate anomaly</a>`;
```

Comment:
I will suggest to link the RCA link to the comment text. In case there are multiple anomalies under the same row, it is easier to distinguish the different anomaly on different days.

Reply:
ok, so should we condense the "Comment" and "link to RCA" columns to one column?

Reply:
Yes. Can be
<a href="https://thirdeye.corp.linkedin.com/app/#/rootcause?anomalyId=1">Comments of anomaly 1</a>
<a href="https://thirdeye.corp.linkedin.com/app/#/rootcause?anomalyId=2">Comments of anomaly 2</a>